src/tools/generate/wrap_tables.js

```diff
@@ -0,0 +1,21 @@
+const jsdom = require("jsdom");
+const { JSDOM } = jsdom;
+
+const wrap_tables = (html) => {
+  const dom = new JSDOM(html);
+
+  const tables = dom.window.document.querySelectorAll('table');
+  [...tables].forEach(table => {
+      const wrapper = dom.window.document.createElement('div');
+      wrapper.className = 'table-wrap';
+
+      table.parentNode.insertBefore(wrapper, table);
+      wrapper.appendChild(table);
+  });
+
+  return dom.window.document.documentElement.outerHTML;;
```

Comment:
I think this is causing html/head/body to be written into the chapter templates.